types/jasmine/ts3.1/index.d.ts

```diff
@@ -155,6 +156,18 @@ interface DoneFn extends Function {
     fail: (message?: Error | string) => void;
 }
 
+/**
+ * Configuration that can be used when configuring Jasmine via {@link jasmine.Env.configure}
+ */
+interface Configuration {
```

Comment:
Could you please rename it to `EnvConfiguration` to not occupy generic words? If later we have any other configuration, it might be nasty to have this one.

Reply:
Fixed in a148df5612